src/flow-spec/IBlock.ts

```diff
@@ -27,14 +27,30 @@ import {
   IContext,
   ICursor,
   IFlow,
-  ISetContactPropertyBlockConfig,
   SetContactProperty,
   ValidationException,
 } from '..'
+import {ISetContactPropertyBlockConfig} from '../model/block/ISetContactPropertyBlockConfig'
```

Comment:
Wrong import, likely to lead to Cyclic Dependencies. import from the index.ts file instead to avoid this possibility.

Reply:
- [ ] import from the index.ts file instead (Fix other files as well)